modules/gltools/src/state-tracker/webgl-parameter-tables.ts

```diff
@@ -327,6 +392,18 @@ export const GL_HOOKED_SETTERS = {
     }),
 
   // SPECIFIC SETTERS
+  useProgram: (update, value) => update({
+    [GL.CURRENT_PROGRAM]: value,
+  }),
+  bindRenderbuffer: (update, target, value) => update({
+    [GL.RENDERBUFFER_BINDING]: value
+  }),
+  bindTransformFeedback: (update, target, value) => update({
```

Comment:
We had to do some nasty workarounds with transform feedback bindings and vertex array bindings as buffers couldn't be bound in two places IIRC. It isn't completely clear to me right now if this was due to issues in Chrome and has subsequently been fixed, or if it was a fundamental OpenGL thing. 

The upshot is that I am unsure of whether caching these fixes things or causes complications. At least we would want to do some careful testing. Maybe keep these as a separate PR?